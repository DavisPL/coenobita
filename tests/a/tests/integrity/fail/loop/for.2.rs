#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::integrity({root}{root})]
    let mut x = 5;

    #[cnbt::integrity({root,b}{root,b})]
    let items = [1, 2, 3];

    for i in items {
        if c::boolean() {
            x = 1;
        }
    }
}