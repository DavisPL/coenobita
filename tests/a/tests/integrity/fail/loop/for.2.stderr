error: expected {root}{root}, found {root}{b,c,root}
  --> tests/integrity/fail/loop/for.2.rs:13:17
   |
13 |             x = 1;
   |                 ^
